modules/tensor_mechanics/src/materials/ADLAROMANCEStressUpdateBase.C

```diff
@@ -533,3 +564,81 @@ ADLAROMCreepStressUpdate<compute_stage>::computePolynomial(const ADReal & value,
     return 2.5 * Utility::pow<3>(value) - 1.5 * value;
   }
 }
+
+template <ComputeStage compute_stage>
+std::vector<std::vector<std::vector<Real>>>
+ADLAROMANCEStressUpdateBase<compute_stage>::getTransformedLimits() const
+{
+  std::vector<std::vector<std::vector<Real>>> transformed_limits(
+      _num_outputs, std::vector<std::vector<Real>>(_num_inputs, std::vector<Real>(2)));
+
+  for (unsigned int i = 0; i < _num_outputs; ++i)
+  {
+    for (unsigned int j = 0; j < _num_inputs; ++j)
+    {
+      for (unsigned int k = 0; k < 2; ++k)
+      {
+        if (_transform[i][j] == ROMInputTransform::EXP)
+          transformed_limits[i][j][k] = std::exp(_input_limits[j][k] / _transform_coefs[i][j]);
+        else if (_transform[i][j] == ROMInputTransform::LOG)
+          transformed_limits[i][j][k] = std::log(_input_limits[j][k] + _transform_coefs[i][j]);
+        else
+          transformed_limits[i][j][k] = _input_limits[j][k];
+      }
+    }
+  }
+
+  return transformed_limits;
+}
+
+template <ComputeStage compute_stage>
+std::vector<std::vector<unsigned int>>
+ADLAROMANCEStressUpdateBase<compute_stage>::getMakeFrameHelper() const
+{
+  std::vector<std::vector<unsigned int>> v(_num_coefs, std::vector<unsigned int>(_num_inputs));
+
+  for (unsigned int numcoeffs = 0; numcoeffs < _num_coefs; ++numcoeffs)
+    for (unsigned int invar = 0; invar < _num_inputs; ++invar)
+      v[numcoeffs][invar] = numcoeffs / MathUtils::pow(_degree, invar) % _degree;
+
+  return v;
+}
+
+template <ComputeStage compute_stage>
```

Comment:
Delete all the methods from here to the end of the file, and instead put `=0` after their declarations in the header file.  There are special situations when you sometimes have to do this sort of thing, but it's not needed here.